app/user/auth-checker-user-only-filter.js

```diff
@@ -1,3 +1,5 @@
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
No need for `{ }` if using `require`. You just need `const Logger = require(...)`

Reply:
{} syntax construct is required or it won't pass yarn test